API.md

```diff
@@ -410,6 +386,12 @@ fast implementation of the ACES sRGB RRT+ODT based on Krzysztof Narkowicz' imple
 #### ToneMaps.ACES\_HILL
 more accurate implementation of the ACES sRGB RRT+ODT based on Stephen Hill's implementation
 
+**Kind**: static property of [<code>ToneMaps</code>](#GltfState.ToneMaps)  
+<a name="GltfState.ToneMaps.NONE"></a>
+
+#### ToneMaps.NONE
+Linear mapping clamped at white
```

Comment:
Nit - it's not actually clamped at white, but per channel at 255. It'll merrily clamp to yellow, cyan or magenta :/

Reply:
Updated.